Update README.Rmd (#13)
steno-aarhus/ukbAid: README.Rmd

the situation described in the point above.

3. Since there are several of us working on this project and since we want to be

> able to collaborate and help each other out the organizing committee for UKB at SDCA

Reviewer: ```suggestion
able to collaborate and help each other out. The organizing committee for UKB at SDCA
```

> will review all project proposals on our UKB Trello page and connect related proejcts.

Reviewer: ```suggestion
will review all project proposals on our UKB Trello page and connect related projects.
```

Since UKB RAP deletes everything when you Terminate the session, you'll be
backing up your project on GitHub. In order to connect to GitHub, you have
to let Git know that you are you (authentication). Like you did on your computer,
run in the Console and paste your secret PAT. If the system still does not recognize

> you, you can additionally setup your username and email:

Reviewer: ```suggestion
you, you probably didn't setup your user name and email with the Git config setup above:
```

```{r}

> gitcreds::gitcreds_set()
> 
> #if necessary add username and email again but uncommenting the code below
> #and adding your own user.name and email
> #ukbAid::setup_git_config("Luke W. Johnston", "lwjohnst@gmail.com")

Reviewer: ```suggestion
# If necessary add username and email again but uncommenting the code below
# and adding your own user.name and email
# ukbAid::setup_git_config("Luke W. Johnston", "lwjohnst@gmail.com")
gitcreds::gitcreds_set()
```

Then, open up the README.md file in your project and follow the instructions
there. When you are ready to create the dataset necessary for your project,
open up the `data-raw/download-data.R` script and follow the instructions there
to create your dataset.

Remember to push your code to your GitHub. Otherwise all your work on the RAP
will be lost next time you login. Also remember to terminate your session when
you are done. 

### Next time you go into UKB RAP

> Next time you open up the RAP you will:

Reviewer: ```suggestion
Next time you open up the RAP you will need to follow these steps below.
```

I am actually updating this section to make it more streamlined (from suggestion by Jonas), so it will be modified shortly :relaxed:
